proto/attest.proto

```diff
@@ -125,10 +125,15 @@ message ContainerState {
   string image_reference = 1;
   bytes image_digest = 2;
   RestartPolicy restart_policy = 3;
+  string image_id = 4;
+  repeated string args = 5;
+  map<string, string> env_vars = 6;
```

Comment:
in the launcher we use a slice of `EnvVar`s. Do we want something similar here (maybe because other metadata will be attached to a specific kv entry) or just leave it as an actual map.

Reply:
A map is fine, no reason to make it overly complicated

Reply:
Ack, thanks

Reply:
Do we need to worry about the order of the env vars here? My intuition is no, so a Map is okay. That means Attestation service may read the Env Var in a different order than it is measured, but I don't that's an issue.

Reply:
I agree that the ordering doesn't seem like an issue here.